sources/sources.bib

```diff
@@ -14,6 +14,31 @@ @inproceedings{sohappy
   series = {PervasiveHealth '17}
 }
 
+@online{tensorflow,
+  author          = {Google},
+  title           = {About TensorFlow},
+  year            = {2020-09-20},
+  url             = {https://www.tensorflow.org/about},
+}
```

Comment:
The `year` attribute usually contains the year the source was released in. If you wanted to write something like "Retrieved September 09, 2020", you should probably add it to a `note` attribute.